steps/tier-ui/search_tier.ts

```diff
@@ -0,0 +1,9 @@
+import { type Page } from '@playwright/test'
+import { Person } from '../delius/utils/person'
+
+export const searchTierByCRN = async (page: Page, crn: string, person: Person) => {
+    await page.getByRole('button', { name: 'Start now' }).click()
+    await page.getByLabel('Search for a person on probation').fill(crn)
```

Comment:
This has changed slightly with the search component changes today
```suggestion
    await page.getByLabel('Find a person on probation').fill(crn)
```

Otherwise the PR looks good